includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -755,6 +761,36 @@ private function has_used_attributes( $attribute_names ) {
 		return true;
 	}
 
+	/**
+	 * Whether a given class is allowed to be styled in <amp-date-picker>.
+	 *
+	 * That component has child classes that won't be present in the document yet.
+	 * So get whether a class is an allowed child.
+	 *
+	 * @since 1.5.0
+	 * @link https://github.com/airbnb/react-dates/tree/05356/src/components
+	 *
+	 * @param string $class The name of the class to evaluate.
+	 * @return bool Whether the class is allowed as a child of <amp-date-picker>.
+	 */
+	private function is_class_allowed_in_amp_date_picker( $class ) {
+		$class_prefixes = [
+			'CalendarDay',
+			'CalendarMonth',
+			'DateInput',
+			'DateRangePicker',
+			'DayPicker',
+			'KeyboardShortcutRow',
+			'SingleDatePicker',
+		];
+
+		if ( preg_match( '#^(' . implode( '|', $class_prefixes ) . ')[a-zA-Z_]+#', $class ) ) {
+			return true;
+		}
+
+		return in_array( $class, [ 'amp-date-picker-calendar-container', 'amp-date-picker-resize-bug' ], true );
```

Comment:
```suggestion
		return 'amp-date-picker-' === substr( $class, 0, 16 );
```

Reply:
Committed in https://github.com/ampproject/amp-wp/pull/4339/commits/eea565f5a78d441428e049073354b40ae94d6399